src/api/app/lib/routes_helper/role_matcher.rb

```diff
@@ -1,14 +1,17 @@
 module RoutesHelper
   class RoleMatcher
     def self.matches?(request)
-      return false if request.bot?
+      login = if ::Configuration.proxy_auth_mode_enabled?
```

Comment:
```suggestion
      return false if request.bot?

      login = if ::Configuration.proxy_auth_mode_enabled?
```

Was the removal of this line (`return false if request.bot?`) intentional?

If so, I would also remove its spec: https://github.com/openSUSE/open-build-service/blob/55fa97dada26341b15017a5fc14b5be3ffae74ca/src/api/spec/lib/routes_helper/role_matcher_spec.rb#L5-L9

I’d actually prefer to keep the bot check.

Reply:
I've mentioned it in the commit message now